charts/yourls/templates/deployment.yaml

```
@@ -130,7 +130,7 @@ spec:
             - name: MARIADB_PORT_NUMBER
               value: {{ include "yourls.databasePort" . | quote }}
             - name: YOURLS_DB_HOST
-              value: {{ include "yourls.databaseHost" . | quote }}
+              value: "{{ include "yourls.databaseHost" . }}:{{ include "yourls.databasePort" . }}"
```

Let's use proper string interpolation:
```suggestion
              value: {{ printf "%s:%s" (include "yourls.databaseHost" .) (include "yourls.databasePort" .) }}
```